site/controllers/search.php

```diff
@@ -3,22 +3,35 @@
 use Kirby\Cms\App;
 
 return function (App $kirby) {
-	$query   = trim(get('q', ''));
-	$area    = trim(get('area', ''));
+	$query = trim(get('q', ''));
+	$area = trim(get('area', ''));
 	$results = null;
 
-	if (empty($query) === false) {
-		$params = [
-			'hitsPerPage'           => (int)get('limit', 50),
-			'attributesToHighlight' => false,
-			'attributesToSnippet'   => '*'
-		];
+	if ($query !== '') {
```

Comment:
`Suggestion:` To make this less nested and easier to read, I am wondering if it would be better to work with an early return, e.g. to check for `if ($query === '') { return [...]  }`.